yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -68,17 +80,53 @@ func runExportDataStatusCmdDbzm() error {
 	}
 	InProgressTableSno := status.InProgressTableSno()
 	var rows []*exportTableMigStatusOutputRow
-	for _, table := range status.Tables {
-		row := &exportTableMigStatusOutputRow{
-			tableName:     table.TableName,
-			status:        "DONE",
-			exportedCount: table.ExportedRowCountSnapshot,
-		}
-		if table.Sno == InProgressTableSno && dbzm.IsLiveMigrationInSnapshotMode(exportDir) {
-			row.status = "EXPORTING"
+	for _, tableStatus := range status.Tables {
+		if withStreamingMode {
+			totalChangesEvents, inserts, updates, deletes, err := metaDB.GetExportedEventsStatsForTable(tableStatus.SchemaName, tableStatus.TableName)
+			row := &exportTableMigStatusOutputRow{
+				schemaName:    tableStatus.SchemaName,
+				tableName:     tableStatus.TableName,
+				exportedCount: tableStatus.ExportedRowCountSnapshot,
+			}
+			if checkCutoverCompleted() {
+				row.status = "DONE"
+			} else {
+				row.status = "STREAMING"
+			}
+			if errors.Is(err, sql.ErrNoRows) {
+				log.Infof("no changes events found for table %s.%s", tableStatus.SchemaName, tableStatus.TableName)
+				row.totalEvents = 0
+				row.numInserts = 0
+				row.numUpdates = 0
+				row.numDeletes = 0
+				row.finalRowCount = tableStatus.ExportedRowCountSnapshot
```

Comment:
Change `MetaDB.GetExportedEventsStatsForTable()` to return `0` for the counter values in case of error.

Then no need to add special handling for the `ErrNoRows` case. Just ignore the error in the first err check itself:
```
if err != nil && !errors.Is(err, sql.ErrNoRows) {
  utils.ErrExit(...)
}
// Move the block from line 107...112 here.
```